data/cfdm_test.pgdump.sql

```diff
@@ -65456,6 +65459,12 @@ ALTER TABLE ONLY sched_i
 ALTER TABLE ONLY sched_i
     ADD CONSTRAINT sched_i_filer_cmte_id_fkey1 FOREIGN KEY (filer_cmte_id) REFERENCES dimcmte(cmte_id);
 
+insert into dimyears values(2012, 2012, CAST('2014-09-03' as timestamp));
```

Comment:
The other changes didn't work so I added the insert statement. Adding the inserts worked so I kept them.